### ISSUE TITLE
Patch release 6.8.1 (suspend setting `enable_starttls_auto`)

### ISSUE THREAD

**Issue by urs-van-binsbergen** [leihs/leihs#1497]
Reminder and Manage mails could not be sent in version 6.8.0, most likely caused by Rails update 6.0/6.1. 
```
Jan 04 03:32:37 zhdk-leihs-prod-v6 leihs-legacy-cron[2868460]: undefined method `disable_starttls_auto' for #<Net::SMTP localhost:25 started=false>
```

The patch solution is to suspend the setting `enable_starttls_auto`  in `legacy/config/initializers/actionmailer_settings.rb`.

**Comment by DrTom** [comment]
https://github.com/rails/rails/pull/44096

**Comment by urs-van-binsbergen** [comment]
We decided on 23.01.2023 to focus on a sustainable solution #1497 which will be published in version 6.9. 
So we won't do this patch.
